duckduckgo/duckduckgo-autofill
# Fade Dax icon in on hover for in-context signup

**Reviewer:** @GioSensation 
**Asana:** https://app.asana.com/0/0/1204087816313067/f

## Description
Swap to colorDax icon on hover

![incontext-signup-icon-hover-2](https://user-images.githubusercontent.com/635903/225948914-5e2d1241-4502-47e2-a2a8-f1eb428cab12.gif)

## Steps to test
1. Build this branch into extension
2. Go to https://duckduckgo.com/newsletter
3. Confirm grayscale icon shows
4. Hover on icon and confirm color icon fades in
